### PR TITLE
Fix build warning

### PR DESCRIPTION
**Pull request by pranavkm**

### CODE REVIEW

File: test/AutobahnTestServer/project.json [hunk 1]
```diff
@@ -1,12 +1,14 @@
...
   "dependencies": {
     "Microsoft.NETCore.Platforms": "1.0.1-*",
     "Microsoft.AspNetCore.WebSockets.Server": "0.1.0-*",
     "Microsoft.AspNetCore.Server.Kestrel": "1.0.0-*"
   },
   "buildOptions": {
-    "emitEntryPoint": true
+    "emitEntryPoint": true,
+    "compile": {
+      "exclude": "wwwroot/**/*"
```
Comment (Tratcher) [inline test/AutobahnTestServer/project.json]: This project doesn't even have a wwwroot dir. You can remove this.

**Comment by JunTaoLuo** [comment]
Are you rebased on dev? I thought I made some of these fixes on dev in https://github.com/aspnet/WebSockets/commit/37931a9fa56d558ab08130783a776fd9fc7a42ee.

**Comment by pranavkm** [comment]
This is the release branch.

**Comment by JunTaoLuo** [comment]
@pranavkm ah that would make sense then. I was thrown off by where you opened the PR on.
image

**Comment by pranavkm** [comment]
Oops,

File: test/AutobahnTestServer/project.json (hunk 1 above)
Comment (JunTaoLuo) [inline test/AutobahnTestServer/project.json]: Agreed. Also remove the redundant `hosting.json` later on in the file. That no longer exists.

**Pull request closed without merging**